sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -622,7 +626,7 @@ def __init__(self, fn, *args, **kwargs):
     super(PTransformWithSideInputs, self).__init__()
 
     if (any([isinstance(v, pvalue.PCollection) for v in args]) or
-        any([isinstance(v, pvalue.PCollection) for v in kwargs.itervalues()])):
+        any([isinstance(v, pvalue.PCollection) for v in itervalues(kwargs)])):
```

Comment:
Can we just do `.values()`?

Reply:
This is a bit less optimal in Python2, but since it's only in the __init__, I'll change it to `.values()`